### PR TITLE
Add Geospatial team community page

### PR DESCRIPTION
**Pull request by fricklerhandwerk**
If you are already existing Geospatial Team [member](https://github.com/NixOS/nixpkgs/blob/master/maintainers/team-list.nix), please let us know if you want/don't want your name to appear in the members list. If you are interested in geospatial software and you are not a team member, please consider joining the team. Thank you !

### CODE REVIEW

File: src/content/teams/13_geospatial.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+The geospatial team is responsible for everything related to geospatial software in [Nixpkgs](https://github.com/NixOS/nixpkgs).
+
+We're keeping widely used packages and modules up to date, and package new software regularly.
+Please ping us if you need help with reviews or build failures.
+
+If you like [what we do](https://github.com/orgs/NixOS/projects/47/views/5?filterQuery=status%3A%22%E2%9C%85+Done%22), please let us know.
+
+If you want to join the geospatial team, add yourself to the geospatial team in the [Nixpkgs team list](https://github.com/NixOS/nixpkgs/blob/master/maintainers/team-list.nix).
+Create PR with short introduction of yourself and assign it to one of the other members.
```
Comment (fricklerhandwerk) [inline src/content/teams/13_geospatial.mdx:28]:
```suggestion
Create pull request with short introduction of yourself and assign it to one of the other members.
```

File: src/content/teams/13_geospatial.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - name: Matrix
+    href: https://matrix.to/#/#geospatial:nixos.org
+  - name: GitHub project board
+    href: https://github.com/orgs/NixOS/projects/47/views/1
+---
+
+The geospatial team is responsible for everything related to geospatial software in [Nixpkgs](https://github.com/NixOS/nixpkgs).
+
+We're keeping widely used packages and modules up to date, and package new software regularly.
+Please ping us if you need help with reviews or build failures.
```
Comment (fricklerhandwerk) [inline src/content/teams/13_geospatial.mdx:23]:
```suggestion
```

File: src/content/teams/13_geospatial.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+members:
+  - name: Ivan Mincik
+    discourse: imincik
+    title: Team lead
+  - name:  Nikolay Korotkiy
+    discourse: sikmir
+    title:
+contact:
+  - name: Discourse
+    href: https://discourse.nixos.org/t/week-in-geospatial-team/37035
```
Comment (imincik) [inline src/content/teams/13_geospatial.mdx:13]:
```suggestion
    href: https://discourse.nixos.org/t/creating-nix-geospatial-team/23454
```

File: src/content/teams/13_geospatial.mdx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  - name: Matrix
+    href: https://matrix.to/#/#geospatial:nixos.org
+  - name: GitHub project board
+    href: https://github.com/orgs/NixOS/projects/47/views/1
```
Comment (imincik) [inline src/content/teams/13_geospatial.mdx:17]:
```suggestion
    href: https://github.com/orgs/NixOS/projects/47
```

File: src/content/teams/13_geospatial.mdx (hunk 3 above)
Comment (fricklerhandwerk) [reply src/content/teams/13_geospatial.mdx:13]:
```suggestion
    href: https://discourse.nixos.org/t/creating-nix-geospatial-team/23454/9
```

That's the post where you make the announcement

**Review by imincik — CHANGES_REQUESTED** [review]

File: src/content/teams/13_geospatial.mdx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+contact:
+  - name: Discourse
+    href: https://discourse.nixos.org/t/creating-nix-geospatial-team/23454/9
+  - name: Matrix
+    href: https://matrix.to/#/#geospatial:nixos.org
+  - name: GitHub project board
+    href: https://github.com/orgs/NixOS/projects/47
+---
+
+The geospatial team is responsible for everything related to geospatial software in [Nixpkgs](https://github.com/NixOS/nixpkgs).
```
Comment (fricklerhandwerk) [inline src/content/teams/13_geospatial.mdx:20]:
```suggestion
The Geospatial Team is responsible for everything related to geospatial software in [Nixpkgs](https://github.com/NixOS/nixpkgs).
```

File: src/content/teams/13_geospatial.mdx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    href: https://github.com/orgs/NixOS/projects/47
+---
+
+The Geospatial Team is responsible for everything related to geospatial software in [Nixpkgs](https://github.com/NixOS/nixpkgs).
+
+We're keeping geospatial packages and modules up to date and we package new software regularly.
+
+If you like [what we do](https://github.com/orgs/NixOS/projects/47), please let us know.
+
+If you want to join the Geospatial Team, add yourself to the geospatial team in the [Nixpkgs team list](https://github.com/NixOS/nixpkgs/blob/master/maintainers/team-list.nix).
```
Comment (fricklerhandwerk) [inline src/content/teams/13_geospatial.mdx:26]:
```suggestion
If you want to join the Geospatial Team, add yourself to the `geospatial` team in the [Nixpkgs team list](https://github.com/NixOS/nixpkgs/blob/master/maintainers/team-list.nix).
```

**Comment by imincik** [comment]
@fricklerhandwerk , thank you very much for helping with this PR. Please wait with merging until we have more feedback from geo team members.

**Comment by willcohen** [comment]
I do want my name to appear!

**Review by thilobillerbeck — APPROVED** [review]

File: src/content/teams/13_geospatial.mdx [hunk 7]
```diff
@@ -0,0 +1,27 @@
+---
+name: Geospatial Team
+description: Making Nix and NixOS the first choice for users of geospatial software
+members:
+  - name: Ivan Mincik
+    discourse: imincik
+    title:
+  - name: Nikolay Korotkiy
+    discourse: sikmir
+    title:
```
Comment (fricklerhandwerk) [inline src/content/teams/13_geospatial.mdx:10]:
```suggestion
    title:
  - name: Will Cohen
    discourse: willcohen
    title:
```

**Comment by imincik** [comment]
@fricklerhandwerk , we are still missing team logo. Now we urgently need to get one ! Do you know who and how did logos for the other teams ?

**Comment by imincik** [comment]
Follow-up PR created https://github.com/NixOS/nixos-homepage/pull/1506

**Review by sikmir — COMMENTED** [review]
It's good for me.

**Pull request merged**